compiler/noirc_evaluator/src/brillig/brillig_ir/debug_show.rs

```diff
@@ -126,6 +126,24 @@ impl DebugShow {
         debug_println!(self.enable_debug_trace, "  MOV {}, {}", destination, source);
     }
 
+    /// Emits a `conditional mov` instruction.
+    pub(crate) fn conditional_mov_instruction(
+        &self,
+        destination: MemoryAddress,
+        source_a: MemoryAddress,
+        source_b: MemoryAddress,
+        condition: MemoryAddress,
+    ) {
+        debug_println!(
+            self.enable_debug_trace,
+            "  {} = CONDITIONAL MOV {} then {}, else {}",
+            destination,
+            condition,
+            source_a,
+            source_b
```

Comment:
```suggestion
        source_then: MemoryAddress,
        source_else: MemoryAddress,
        condition: MemoryAddress,
    ) {
        debug_println!(
            self.enable_debug_trace,
            "  {} = CONDITIONAL MOV {} then {}, else {}",
            destination,
            condition,
            source_then,
            source_else
```
I'd consider putting an `if` in there as well. Perhaps `MOV_IF` like `JUMP_IF`?

Reply:
Done